consensus/istanbul/core/handler.go

```diff
@@ -156,7 +156,17 @@ func (c *core) handleMsg(payload []byte) error {
 	msg := new(message)
 	if err := msg.FromPayload(payload, c.validateFn); err != nil {
 		if c.backend.NodeType() == common.CONSENSUSNODE {
-			logger.Error("Failed to decode message from payload", "err", err)
+			if err != istanbul.ErrUnauthorizedAddress {
+				logger.Error("Failed to decode message from payload", "err", err)
+			} else {
+				// Decode msg to derive view, so node's view and msg's view can be compared
+				msgView, msgDecodeErr := msg.GetView()
+				if msgDecodeErr != nil {
+					logger.Error("Failed to decode message", "code", msg.Code, "err", msgDecodeErr)
+					return errInvalidMessage
+				}
+				logger.Warn("Signed by an unauthorized address. If syncing, it may or may not be an unauthorized address.", "node view", c.currentView().String(), "msg view", msgView.String())
```

Comment:
The sentence is too ambiguous 
```suggestion
				logger.Warn("Signed by an unauthorized address. It could happen when there is an unsynced validator in the network", "node view", c.currentView().String(), "msg view", msgView.String())
```

Reply:
@aidan-kwon Thanks, it's applied at https://github.com/klaytn/klaytn/pull/1397/commits/458d7e27ef990c721f8a4adc1177de1518415d87, and i add some comments to explain more specific. 

Also, you suggested to print sender address at `CheckValidatorSignature`, but i think adding sender address at warning log here. Can you take a look?

Reply:
I also agree with @yoomee1313's opinion. Printing msg view and sender address information here will sufficiently help us determine whether the warning is caused by unauthorized sender or the unsynced node. 
@aidan-kwon Would you mind printing the sender address here?

Reply:
I thought an additonal log message which prints both a message sender and a validator list in `CheckValidatorSignature` more easily. However, this change is sufficient as well.